tesseract-core/src/schema.rs

```diff
@@ -138,6 +141,83 @@ pub struct Cube {
     pub annotations: Option<Vec<Annotation>>,
 }
 
+impl Cube {
```

Comment:
These cube methods have a good feel, and reflect how I think I'll be moving some `Schema` functionality to `Cube`